style_guides/accessibility_guide.md

```diff
@@ -0,0 +1,108 @@
+# Accessibility (A11Y) Guide
+
+This document provides some technical guidelines how to prevent several common
+accessibility issues.
+
+## Interactive elements
+
+### Use `<button>` and `<a>`
+
+**TL;DR** *Use `<button>` and `<a>` instead of click listeners on other elements
+and style it whatever way you need.*
+
+If you want to make an element clickable, use a `<button>` or `<a>` element for it.
+Use a *button* whenever it causes an action on the current page, and an *a* if it
+navigates to a different page. You can use click listeners just fine on these elements.
```

Comment:
Isn't it non-semantic to use a link as a button? I think links should be used for navigating to different URLs, and buttons should be used for triggering event handlers. That way you know that you can open links in new windows or tabs, or just reference the URL it points to.